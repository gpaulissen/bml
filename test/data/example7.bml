2C Strong, forcing
  2D Waiting
2X Weak
  2N Ogust
