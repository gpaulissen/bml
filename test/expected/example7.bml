#VUL 00

#SEAT 0

2C = Strong, forcing
  2D = Waiting
2D = Weak
  2N = Ogust
2H = Weak
  2N = Ogust
2S = Weak
  2N = Ogust
