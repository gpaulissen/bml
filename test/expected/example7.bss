*00{}=NYYYYYY
002C=NYYYYYY008Strong, forcing
002CP2D=NYYYYYY008Waiting
002D=NYYYYYY008Weak
002DP2N=NYYYYYY0Ogust
002H=NYYYYYY008Weak
002HP2N=NYYYYYY0Ogust
002S=NYYYYYY008Weak
002SP2N=NYYYYYY0Ogust
