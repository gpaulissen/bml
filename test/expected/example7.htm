<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Strict//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd'>
<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
  <title>
   No title supplied
  </title>
  <style type="text/css">
   div.start {
    display: inline-block;
    vertical-align: top;
    width: 60px;
}
    
div.bidtable {
    width:700px;
    padding-bottom: 20px;
}    

div.bidtable ul {
    list-style-type: none;
    margin: 0;
    padding: 0;
}

div.bidtable li ul
{
    list-style-type: none;
    margin: 0;
    padding: 0;
    margin-left: 60px;
}

div.tree {
    list-style-type: none;
    padding-bottom: 20px;
}

div.tree ul {
    list-style-type: none;
    /* vline.png */
    background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAAKAQMAAABPHKYJAAAAA1BMVEWIiIhYZW6zAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH1ggGExMZBky19AAAAAtJREFUCNdjYMAEAAAUAAHlhrBKAAAAAElFTkSuQmCC);
    background-size: 2px;
    background-repeat: repeat-y;
    margin: 0;
    padding: 0;
    margin-left: 10px;
}

div.tree li.node {
    margin: 0;
    padding: 0 12px;
    line-height: 20px;
    /* node.png */
    background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAUAQMAAACK1e4oAAAABlBMVEUAAwCIiIgd2JB2AAAAAXRSTlMAQObYZgAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9YIBhQIJYVaFGwAAAARSURBVAjXY2hgQIf/GTDFGgDSkwqATqpCHAAAAABJRU5ErkJggg==);
    background-size: 12px;
    background-repeat: no-repeat;
}

div.tree td.node {
    vertical-align: top;
    width: 60px;
}

div.tree li.last {
    /* lastnode.png */
    background-color: #fff;
    background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAUAQMAAACK1e4oAAAABlBMVEUAAwCIiIgd2JB2AAAAAXRSTlMAQObYZgAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9YIBhQIIhs+gc8AAAAQSURBVAjXY2hgQIf/GbAAAKCTBYBUjWvCAAAAAElFTkSuQmCC) ;
    background-size: 12px;
    backrgound-repeat: no-repeat;
}

div.tree li.root {
    background-color: #fff;
    background-image: none;
    backrgound-repeat: no-repeat;
}

.ccolor{color:Green}
.dcolor{color:Orange}
.hcolor{color:Red}
.scolor{color:Black}

@media print {
    * {
        -webkit-print-color-adjust: exact !important;   /* Chrome, Safari */
				color-adjust: exact !important;                 /* Firefox */
		}
    div.tree ul {
        background-size: 1px;
    }
}
  </style>
 </head>
 <body>
  <div class="bidtable">
   <ul>
    <li>
     <div class="start">
      &empty;
     </div>
     <ul>
      <li>
       <div class="start">
        2
        <span class="ccolor">
         &clubs;
        </span>
       </div>
       Strong, forcing
       <ul>
        <li>
         <div class="start">
          2
          <span class="dcolor">
           &diams;
          </span>
         </div>
         Waiting
        </li>
       </ul>
      </li>
      <li>
       <div class="start">
        2X
       </div>
       Weak
       <ul>
        <li>
         <div class="start">
          2NT
         </div>
         Ogust
        </li>
       </ul>
      </li>
     </ul>
    </li>
   </ul>
  </div>
 </body>
</html>